crates/mun_runtime/src/lib.rs

```diff
@@ -271,6 +246,22 @@ impl Runtime {
         }
         false
     }
+
+    /// Returns the runtime's allocator.
+    pub(crate) fn gc(&self) -> &Arc<GarbageCollector> {
+        &self.gc
+    }
+
+    /// Collects all memory that is no longer referenced by rooted objects. Returns `true` if memory
+    /// was reclaimed, `false` otherwise.
+    pub fn collect(&self) -> bool {
```

Comment:
```suggestion
    pub fn gc_collect(&self) -> bool {
```

This is a little more verbose, especially since the `collect` function name is already commonly used throughout Rust codebases, for an entirely different purpose.